src/core/factcheck_plus/policy_scorer.py

```
@@ -0,0 +1,273 @@
+        """Wendet die harten Gates an; gibt einen Ausschluss-Status oder ``None``
+        (= eligible) zurück. ``None`` bedeutet: geht ins Scoring/Quotenauswahl."""
+        claim_type = claim.refined.claim_type
+        role = claim.mapping.argument_role
+        claim_class = ROLE_TO_CLASS.get(role, ClaimClass.C)
+
+        # Gate 1 — Meinung/Deutung: gar nicht recherchieren.
+        if claim_type in self._gates.get("exclude_claim_types", []):
+            return STATUS_EXCLUDED_OPINION
+
+        # Gate 3 — Basisfakt (Metadaten-Rolle) ODER trivial: nur listen, nicht
+        # recherchieren. Trivialität = (Skalenmax − non_triviality); ab
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)